spec/default_facts.yml

```diff
@@ -2,7 +2,8 @@
 #
 # Facts specified here will override the values provided by rspec-puppet-facts.
 ---
-ipaddress: "172.16.254.254"
-ipaddress6: "FE80:0000:0000:0000:AAAA:AAAA:AAAA"
+networking:
+    ip: "172.16.254.254"
+    ip6: "FE80:0000:0000:0000:AAAA:AAAA:AAAA"
+    mac: "AA:AA:AA:AA:AA:AA"
 is_pe: false
```

Comment:
Isn't the `is_pe` fact dead?

Reply:
hey @ekohl, this fact will be removed in the templates shortly.